assets/backport/github/github.go

```diff
@@ -85,20 +89,20 @@ func (c *Client) Backport(ctx context.Context, baseBranchName string, pullNumber
 	// Create a new branch off of the target branch.
 	err := c.createBranchFrom(ctx, baseBranchName, newBranchName)
 	if err != nil {
-		return "", trace.Wrap(err)
+		return "", fmt.Errorf("failed to create new branch %s: %w", newBranchName, err)
```

Comment:
Should we keep using `trace` for errors? Maybe use `trace.Errorf` or `trace.WrapWithMessage` or create the appropriate error type (e.g., `trace.BadParameter`)?